zerver/forms.py

```diff
@@ -466,13 +466,9 @@ def clean(self) -> Dict[str, Any]:
                     subdomain,
                     return_data.get("matching_user_ids_in_different_realms"),
                 )
-                error_message = _(
-                    "Your Zulip account {username} is not a member of the "
-                    + "organization associated with this subdomain.  "
-                    + "Please contact your organization administrator with any questions."
-                )
-                error_message = error_message.format(username=username)
-                raise ValidationError(error_message)
```

Comment:
Removal of this error message that leaks some information was requested here - https://github.com/zulip/zulip/pull/13924#discussion_r676051716